src/lib/formats/coco_rawdsk.cpp

```diff
@@ -4,30 +4,56 @@
 
     CoCo Raw Disk
 
+	Anonymous format for internal use by CoCo file systems
+
 ***************************************************************************/
 
 #include "coco_rawdsk.h"
 
 
+//-------------------------------------------------
+//  ctor
+//-------------------------------------------------
+
 coco_rawdsk_format::coco_rawdsk_format() : wd177x_format(formats)
 {
 }
 
+
+//-------------------------------------------------
+//  name
+//-------------------------------------------------
+
 const char *coco_rawdsk_format::name() const
 {
-	return "coco_rawdsk";
+	return nullptr;	// "raw" disk is anonymous
 }
 
+
+//-------------------------------------------------
+//  description
+//-------------------------------------------------
+
 const char *coco_rawdsk_format::description() const
 {
-	return "CoCo Raw Disk";
+	return nullptr;	// "raw" disk is anonymous
 }
 
+
+//-------------------------------------------------
+//  extensions
+//-------------------------------------------------
+
 const char *coco_rawdsk_format::extensions() const
 {
-	return "raw";
+	return nullptr;	// "raw" disk is anonymous
 }
 
+
+//-------------------------------------------------
+//  formats
+//-------------------------------------------------
+
 const coco_rawdsk_format::format coco_rawdsk_format::formats[] =
```

Comment:
Seriously, stop adding zero-value comments like this.  A comment that just repeats the name of the variable/function is useless clutter.